### DISCUSSION TITLE
Single Rig and Session schemas

### DISCUSSION THREAD

**Discussion by saskiad** [AllenNeuralDynamics/aind-data-schema#424] [Ideas]
We've discussed the idea of having a single rig and single session schema (as opposed to separate ephys/ophys/behavior schemas for these). There is a lot of shared devices across these rigs and with plans to do experiments that combine modalities, having a single schema makes sense.
One concern I have is that there are devices required for an ephys or ophys rig that would need to be optional in a generic rig. So we might need to lean more into validators to make sure we're still getting the information we need/expect for each modality.

thoughts? @dyf

**Comment by dyf** [comment]
Cleaning up the discussion board.  We're doing this!
